splunk_add_on_ucc_framework/install_python_libraries.py

```diff
@@ -152,3 +162,65 @@ def remove_execute_bit(installation_path: str) -> None:
             logger.info(f"  fixing {o} execute bit")
             current_permissions = stat.S_IMODE(os.lstat(o).st_mode)
             os.chmod(o, current_permissions & no_exec)
+
+
+def install_os_dependent_libraries(
+    package_path: str, ucc_lib_target: str, installer: str
+) -> None:
+    conf_file_name = "os-dependent_packages.json"
+    config_file = os.path.join(package_path, "lib", conf_file_name)
+
+    if not os.path.isfile(config_file):
+        logger.info(
+            "Config file not found. Installation of os-dependent libraries skipped."
+        )
+        return
+
+    try:
+        with open(config_file) as file:
+            packages = json.load(file)["libraries"]
+    except (KeyError, ValueError) as e:
+        logger.error(
+            f"Invalid json structure. Please verify content of {conf_file_name} file. "
+            f"Error: {repr(e)}"
+        )
+        return
```

Comment:
Should we fail here? If the file is not correct then the build step should not succeed I think.

Reply:
good point, I will change it